sklearn/linear_model/least_angle.py

```diff
@@ -587,14 +587,15 @@ class Lars(LinearModel, RegressorMixin):
     sklearn.decomposition.sparse_encode
 
     """
+    method = 'lar'
```

Comment:
I know it is done in other part of the code but is it absolutely necessary to put it here ?